packages/react-native-reanimated/src/reactUtils.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function getCurrentReactOwner() {
+  // @ts-expect-error React secret internals aren't typed
+  return React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED
```

Comment:
I hope I won't get fired because of using this secret variable

Reply:
We'll find out 😈